src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -393,27 +452,47 @@ def __get_pydantic_json_schema__(
         return json_schema
 
 
-class ClassMeta(BaseModel):
+class MetadataBase(BaseModel):
```

Comment:
Renamed this to reflect that these components are the base top-level fields in our current two metadata cases: case metadata and object metadata